units/crons.go

```diff
@@ -109,17 +109,24 @@ func PopulateEventAlertProcessing(env evergreen.Environment) amboy.QueueOperatio
 			return nil
 		}
 
-		err = dispatchUnprocessedNotifications(ctx, queue, flags)
-		grip.Error(message.WrapError(err, message.Fields{
-			"message": "unable to dispatch unprocessed notifications",
-		}))
-		notifySettings := env.Settings().Notify
-		if err = notifySettings.Get(env); err != nil {
-			return errors.Wrap(err, "unable to get notify settings")
+		return dispatchUnprocessedNotifications(ctx, queue, flags)
+	}
+}
+
+func PopulateEventNotifierJobs(env evergreen.Environment) amboy.QueueOperation {
+	return func(ctx context.Context, queue amboy.Queue) error {
+		flags, err := evergreen.GetServiceFlags()
+		if err != nil {
+			return errors.WithStack(err)
 		}
-		limit := notifySettings.EventProcessingLimit
-		if limit <= 0 {
-			limit = evergreen.DefaultEventProcessingLimit
+
+		if flags.EventProcessingDisabled {
+			grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
+				"message": "alerts disabled",
+				"impact":  "not processing alerts for notifications",
```

Comment:
Can this message be different than the one for PopulateEventSendJobs?

Reply:
Yes, fixed.